src/App.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { useSelector, useDispatch } from "react-redux";
 import { BrowserRouter, Route, Switch, Link, Redirect } from "react-router-dom";
-import { FullScreen, useFullScreenHandle } from "react-full-screen";
+//import { FullScreen, useFullScreenHandle } from "react-full-screen";
```

Comment:
need to see if there is a way to dynamically disable fullscreen for cypress testing